tools/docker/README.md

```diff
@@ -442,55 +442,6 @@ You’ll likely need to install a browser extension like the following:
 * [The easiest Xdebug](https://addons.mozilla.org/en-US/firefox/addon/the-easiest-xdebug/) for Mozilla Firefox
 * [Xdebug Helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc) for Google Chrome
 
-#### Remote debugging with Atom editor
```

Comment:
Looking at the results of a little poll, that seems like a good idea. No active contributors use Atom anymore.

Reply:
Awesome, thank you for conducting this survey @jeherve ❤️